app/templates/admin/userManagement.html

```diff
@@ -92,32 +92,23 @@ <h3 class="accordion-header" id="headingThree">
         </button>
       </h3>
       <div id="collapseThree" class="accordion-collapse collapse" aria-labelledby="headingThree" data-bs-parent="#adminManagement">
-        <div class="accordion-body">
+        <div id="programSelect" form="programSettings" class="accordion-body programData">
```

Comment:
I don't think any of these attributes are necessary anymore. Are they?

Reply:
We are going to look at the display responsivity later